source/quick-start-rails/view-data.txt

```diff
@@ -0,0 +1,90 @@
+.. _mongoid-quick-start-rails-view-data:
+
+=================
+View MongoDB Data
+=================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create a data model
+
+      Run the following command from your project root to create a
+      ``Restaurant`` model with ``name``, ``cuisine``, and ``borough``
+      fields:
+
+      .. code-block:: bash
+
+         bin/rails g scaffold Restaurant name:string cuisine:string borough:string
```

Comment:
I have never seen CLI class creation before 😮

Reply:
common for ODMs! [laravel](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/quick-start/view-data/#create-a-model-and-controller)